docs/guides/cluster-to-advanced-cluster-migration-guide.md

```diff
@@ -0,0 +1,175 @@
+# Cluster to Advanced Cluster Migration Guide
+
+**Objective**: Guide users to replace the `mongodbatlas_cluster` resource with the `mongodbatlas_advanced_cluster` resource.
+
+-> **NOTE:** This guide focus on the resource migration as the data source migration is only requiring a resource_type change from `data.mongodbatlas_cluster` to `data.mongodbatlas_advanced_cluster`.  However, pay attention to the [output changes.](#output-changes)
+
+## Main Changes Between `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster`
+
+Key differences in the configuration:
```

Comment:
Nit: this list heading seems a bit redundant (basically the same as the section heading above?)

Reply:
Agree. Removed